content/docs/2.0/scalers/apache-kafka.md

```diff
@@ -34,21 +33,25 @@ triggers:
 
 ### Authentication Parameters
 
- You can use `TriggerAuthentication` CRD to configure the authenticate by providing authMode, username, password. If your kafka cluster does not have sasl authentication turned on, you will not need to pay attention to it.
+ You can use `TriggerAuthentication` CRD to configure the authenticate by providing `sasl`, `username` and `password`, in case your Kafka cluster has SASL authentication turned on. If TLS is required you should set `tls` to `enabled` and provide `ca`, `cert` and `key`.
 
 **Credential based authentication:**
 
-- `authMode` Kafka sasl auth mode. Optional. The default value is none. For now, it must be one of none, sasl_plaintext, sasl_ssl, sasl_ssl_plain, sasl_scram_sha256, sasl_scram_sha512.
-- `username` Optional. If authmode is not none, this is required.
-- `password` Optional.If authmode is not none, this is required.
-- `ca` Certificate authority file for TLS client authentication. Optional. If authmode is sasl_ssl, this is required.
-- `cert` Certificate for client authentication. Optional. If authmode is sasl_ssl, this is required.
-- `key` Key for client authentication. Optional. If authmode is sasl_ssl, this is required.
+SASL:
+- `sasl`: Kafka SASL auth mode. Optional. If not set, SASL for Kafka is not used. If set, it must be one of `plaintext`, `scram_sha256` or `scram_sha512`.
+- `username`: Optional. If `sasl` is set, this is required.
+- `password`: Optional. If `sasl` is set, this is required.
+
+TLS:
+- `tls`: Optional. To enable SSL auth for Kafka, set this to `enable`. If not set, TLS for Kafka is not used.
```

Comment:
I am not sure, what is the best way to handle this, whether this apporach, with `tls: enable`, or somehing like `enableTls: true/false` ? I have chosen the current approach to have the config similar to SASL one

Reply:
I like just `tls` :-)

Reply:
I think for others we use "use" or "enable".

It you read a spec and it just says "tls", what does it mean? Nothing beats a bool with a user friendly name

Reply:
@tomkerkhove well, TLS is kinda selfexplanatory and as I said I kinda like the consistency between the properties, ie. `sasl`, `tls`

Reply:
Fine by me, just wanted to elaborate on my thinking